Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM frolvlad/alpine-python3
```

Comment:
don't know if that would work here, but did @tarekziade have a special python slim img here for us to use for molotov-based tests?